openwisp_utils/admin_theme/system_info.py

```diff
@@ -0,0 +1,55 @@
+import platform
+from collections import OrderedDict
+
+import pkg_resources
+from django.conf import settings
+from django.utils.module_loading import import_string
+
+EXTRA_OPENWISP_PACKAGES = ['netdiff', 'netjsonconfig']
+
+
+def get_installed_openwisp_packages():
+    dists = pkg_resources.working_set
+    return {
+        dist.key: dist.version
+        for dist in dists
+        if dist.key.startswith('openwisp') or dist.key in EXTRA_OPENWISP_PACKAGES
+    }
+
+
+def get_openwisp_version():
+    try:
+        return import_string('openwisp2.__openwisp_version__')
+    except ImportError:
+        return None
+
+
+def get_enabled_openwisp_modules():
+    enabled_packages = {}
+    installed_packages = get_installed_openwisp_packages()
+    extra_packages = {}
+    for package, version in installed_packages.items():
+        if package in EXTRA_OPENWISP_PACKAGES:
+            extra_packages[package] = version
+            continue
+        package_name = package.replace("-", "_")
```

Comment:
why are you using double quotes here? We generally use single quotes around the codebase.